Solutions/IoTOTThreatMonitoringwithDefenderforIoT/Analytic Rules/IoTUnauthorizedPLCModifications.yaml

```diff
@@ -79,6 +79,7 @@ query: |
   or AlertName == "Unauthorized PLC Configuration Write"
   or AlertName == "Unauthorized PLC Program Upload"
   or ExtendedProperties contains "Illegal Beckhoff AMS Command"
+  or ExtendedProperties contains "Beckhoff AMS Command Failure"
```

Comment:
Why are we using `contains` instead of `has` in both line 81 and 82?  Also, I would suggest for lines 22 thru 82 you use a [dynamic](https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/scalar-data-types/dynamic) list for the AlertName matching that is currently an embedded list in the query and then use [`has_any`](https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/has-anyoperator#use-has_any-operator-with-a-dynamic-array) against that dynamic list.

Reply:
@shainw Thanks! Changes done. could you please review?

Reply:
@shainw what's the next step?